enhancements/build-as-userns.md

```diff
@@ -0,0 +1,82 @@
+# Build containers images as user
+
+## Goals
+
+There is two different scenarios we want to achieve :
+
+1) Allow user to run buildah as root inside the container but running as user on the host
+
+2) Running as user inside the container.
+
+### Scenario 1 Running as root on container as user on host
+
+* Openshift 4.10 will introduces a new "builder" workload on cri-o config. This
+  allows running container as user on the host but shows as root on host. The
```

Comment:
typo: "running container as user on the host but shows as root on the container"